pandas/core/groupby.py

```diff
@@ -118,12 +120,12 @@
     Examples
     --------
     {examples}
-
     See also
     --------
```

Comment:
can you reference Series/DataFrame.apply as well

Reply:
Could you add these to the See Also?